docs/acme-divergences.md

```diff
@@ -0,0 +1,82 @@
+# Boulder divergences from ACME
+
+While Boulder attempts to implement the ACME specification as strictly as possible
+there are places at which we will diverge from the letter of the specification for
+various reason.
+
+This document details these differences, since ACME is not yet finalized it will be
+updated to be up to date with the most recent numbered draft that has been published.
```

Comment:
I think part of the clause after the comma is redundant.

Maybe change "updated to be up to date with the most recent numbered draft that has been published" -> "updated as numbered drafts are published" ?